pyproject.toml

```diff
@@ -2,7 +2,7 @@
 name = "qat-compiler"
 # This name has the -compiler suffix in order to use the poetry and twine tools to build and publish to PyPI
 # witout having to manually adjust the dist file names.
-version = "2.0.3"
+version = "2.1.0"
```

Comment:
Not critical but we should adjust the authors list

Reply:
Done (assuming I didn't miss anyone)